arm-graphrbac/1.6/swagger/graphrbac.json

```diff
@@ -2438,7 +2555,46 @@
         "includeDirectoryObjectReferences"
       ],
       "description": "Request parameters for the GetObjectsByObjectIds API."
-    }      
+    },
+    "Domain": {
+      "properties": {
+        "authenticationType": {
+          "type": "string",
+          "description": "the type of the authentication into the domain.",
+          "readOnly": true
+        },
+        "isDefault": {
+          "type": "boolean",
+          "description": "if this is the default domain in the tenant.",
+          "readOnly": true
+        },
+        "isVerified": {
+          "type": "boolean",
+          "description": "if this domain's ownership is verified.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "the domain name."
+        }
+      },
+      "required": [
```

Comment:
Seems like `Domain` is only used in the `DomainListResult` and in that case `"required"` should not be used for `name` and it'd be good to mark it as readOnly.